testflo/main.py

```diff
@@ -131,7 +131,9 @@ def dir_exclude(d):
                                     dir_exclude=dir_exclude)
         benchmark_file = open(options.benchmarkfile, 'a')
     else:
-        discoverer = TestDiscoverer(dir_exclude=dir_exclude)
+        discoverer = TestDiscoverer(dir_exclude=dir_exclude,
+                                    func_pattern=six.text_type(options.test_glob),
```

Comment:
I added that to better match `nose`'s "-m" parameter which reads "Files, directories, function names, and class names that match this regular expression are considered tests." Though nose supports general regex instead of fnmatch.